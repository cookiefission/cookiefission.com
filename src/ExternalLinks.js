import React from 'react';

import GithubLink from './links/GithubLink';
import TwitterLink from './links/TwitterLink';
import MediumLink from './links/MediumLink';

const ExternalLinks = () => {
  return(
    <div className='ExternalLinks' >
      <GithubLink />
      <TwitterLink />
      <MediumLink />
    </div>
  );
};

export default ExternalLinks;
