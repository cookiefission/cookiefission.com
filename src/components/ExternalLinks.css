.ExternalLinks {
  display: flex;
  justify-content: space-around;
  max-width: 200px;
  margin: auto;
}
