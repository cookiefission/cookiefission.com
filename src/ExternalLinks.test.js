import React from 'react';
import ExternalLinks from './ExternalLinks';
import renderer from 'react-test-renderer';

describe('<ExternalLinks />', () => {
  it('matches the snapshot', () => {
    const component = renderer.create(<ExternalLinks />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
