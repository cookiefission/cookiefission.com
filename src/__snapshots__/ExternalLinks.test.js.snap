exports[`<ExternalLinks /> matches the snapshot 1`] = `
<div
  className="ExternalLinks">
  <a
    href="https://github.com/cookiefission"
    target="blank">
    <div
      className="Icon-wrapper">
      <i
        className="fa fa-github fa-lg" />
    </div>
  </a>
  <a
    href="https://twitter.com/cookiefission"
    target="blank">
    <div
      className="Icon-wrapper">
      <i
        className="fa fa-twitter fa-lg" />
    </div>
  </a>
  <a
    href="https://blog.cookiefission.com"
    target="blank">
    <div
      className="Icon-wrapper">
      <i
        className="fa fa-medium fa-lg" />
    </div>
  </a>
</div>
`;
